package.json

```
@@ -15,15 +15,15 @@
     "build-js:device-intelligence": "mkdir -p dist/public/javascripts; uglifyjs node_modules/@govuk-one-login/frontend-device-intelligence/build/esm/index.js -o dist/public/javascripts/deviceIntelligence.js",
     "build:js:all": "mkdir -p dist/public/javascripts; uglifyjs node_modules/govuk-frontend/govuk/all.js node_modules/hmpo-components/all.js --beautify -o dist/public/javascripts/all.js",
     "minfiy-build-js": "uglifyjs src/assets/javascript/application.js -o src/assets/javascript/application.js -c -m && uglifyjs src/assets/javascript/cookies.js -o src/assets/javascript/cookies.js -c -m",
-    "build:sass": "rm -rf dist/public/stylesheets/application.css && sass --no-source-map src/assets/scss/application.scss dist/public/stylesheets/application.css --style compressed && sass --no-source-map node_modules/@govuk-one-login/frontend-ui/build/all.css dist/public/stylesheets/frontend-ui.css --style compressed",
-    "copy-assets": "mkdir -p dist && copyfiles -u 1 src/**/*.njk dist/ src/locales/**/** dist/ && copyfiles -u 3 src/assets/javascript/*.js dist/public/scripts && copyfiles -u 3 src/assets/images/** dist/public/images && cp -R node_modules/@govuk-one-login/frontend-ui/build/frontendUiAssets dist/public/",
+    "build:sass": "rm -rf dist/public/stylesheets/application.css && sass --no-source-map src/assets/scss/application.scss dist/public/stylesheets/application.css --style compressed",
+    "copy-assets": "mkdir -p dist && copyfiles -u 1 src/**/*.njk dist/ src/locales/**/** dist/ && copyfiles -u 3 src/assets/javascript/*.js dist/public/scripts && copyfiles -u 3 src/assets/images/** dist/public/images",
     "check-translation": "node node_modules/@govuk-one-login/di-ipv-cri-common-express/scripts/checkTranslations.js ../../../../src/locales/",
     "test:coverage": "npm run test -- --coverage",
     "test": "jest"
   },
   "dependencies": {
     "@aws-sdk/client-dynamodb": "3.668.0",
-    "@govuk-one-login/di-ipv-cri-common-express": "11.0.1",
+    "@govuk-one-login/di-ipv-cri-common-express": "^11.0.1",
```

I guess npm might have unpinned this dep?

```suggestion
    "@govuk-one-login/di-ipv-cri-common-express": "11.0.1",
```

---

@MarieseMikely if you can make this change I think it'll be good to go